lib/agent/reporter.rb

```diff
@@ -46,6 +64,7 @@ def create_reporter_thread
             send_report_batch unless @event_cache.num_events.zero?
           end
         end
+        report_thread.priority = 255
 
         at_exit do
           @report_queue.close
```

Comment:
Turns out this doesn't exist on Ruby <= 2.3. See #54 for working (failing) tests showing this. 😢

Reply:
😢 Well I guess that makes any argument for keeping it moot :)